crowdin/groups.go

```diff
@@ -76,3 +76,101 @@ func (s *GroupsService) Edit(ctx context.Context, id int, req []*model.UpdateReq
 func (s *GroupsService) Delete(ctx context.Context, id int) (*Response, error) {
 	return s.client.Delete(ctx, fmt.Sprintf("/api/v2/groups/%d", id), nil)
 }
+
+// List returns a list of managers.
```

Comment:
It would be better to place the group manager methods in the `users` resource, according to the API docs structure